backend/api/adjudicationsApi.ts

```diff
@@ -0,0 +1,48 @@
+import { OauthApiClient } from './oauthEnabledClient'
+import contextProperties from '../contextProperties'
+import { mapToQueryString } from '../utils'
+
+export const adjudicationsApiFactory = (client: OauthApiClient) => {
+  const processResponse = (context) => (response) => {
+    contextProperties.setResponsePagination(context, response.headers)
+    return response.body
+  }
+
+  const get = (context, url, resultsLimit?, retryOverride?) =>
+    client.get(context, url, { resultsLimit, retryOverride }).then(processResponse(context))
+
+  const getAdjudications = async (context, offenderNumber, params, pageOffset, pageLimit) => {
+    const paramsWithPageAndSize = {
+      ...params,
+      page: pageOffset / pageLimit,
+      size: pageLimit,
+    }
+
+    const response = await get(
+      context,
+      `/adjudications/${offenderNumber}/adjudications${params && `?${mapToQueryString(paramsWithPageAndSize)}`}`
```

Comment:
[Here is this endpoint](https://github.com/ministryofjustice/hmpps-manage-adjudications-api/blob/9e4fb75fe356845eda7c96c66aab998f792aecee/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/AdjudicationSummaryController.kt#L142-L208) in the AdjudicationsAPI

It now provides the pagination information in `results` see [SummaryAdjudicationService.kt](https://github.com/ministryofjustice/hmpps-manage-adjudications-api/blob/fc46931a8d940ffe3d2f14b497fc6d23f81212fa/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/SummaryAdjudicationService.kt#L53-L60) hence I've updated `pagination` and the tests